packages/@sanity/code-input/src/schema.tsx

```diff
@@ -1,87 +1,18 @@
-import React, {PureComponent} from 'react'
+import React from 'react'
 import CodeBlockIcon from 'part:@sanity/base/code-block-icon'
-import JsonLogo from './jsonLogo'
-
-class Input extends PureComponent<import('./CodeInput').CodeInputProps> {
-  CodeInput: typeof import('./CodeInput').default
-  codeInput: React.RefObject<import('./CodeInput').default>
-
-  constructor(props: any) {
-    super(props)
-
-    this.CodeInput = require('./CodeInput').default
-    this.codeInput = React.createRef()
-  }
-
-  focus() {
-    if (this.codeInput && this.codeInput.current) {
-      this.codeInput.current.focus()
-    }
-  }
-
-  render() {
-    const {CodeInput} = this
-
-    return <CodeInput ref={this.codeInput} {...this.props} />
-  }
-}
-
-const Preview = (props: import('./PreviewCode').PreviewCodeProps) => {
-  const PreviewCode = require('./PreviewCode').default
+import CodeInput from './CodeInput'
+import PreviewCode, {PreviewCodeProps} from './PreviewCode'
+import {getMedia} from './getMedia'
 
+const Preview = (props: PreviewCodeProps) => {
   return <PreviewCode {...props} />
 }
```

Comment:
if this code no longer does that `require(/* ... */).default`, can you inline the `PreviewCode` component in the `extendedPreview` below?